Cargo.toml

```diff
@@ -16,7 +16,7 @@ libc = "0.2.88"
 local_ipaddress = "0.1.3"
 lazy_static = "1.4.0"
 
-[build-dependencies]
+[target.'cfg(target_os = "android")'.build-dependencies]
```

Comment:
That's not going to work, since the build script is compiled and run on the host machine and it is a dependency of the build script.

Reply:
Haha I also had a hard time wrapping my head around this.